src/core_atmosphere/Registry.xml

```diff
@@ -1311,15 +1314,250 @@
+                        <var name="ru"/>
+                        <var name="ru_p"/>
+                        <var name="rw"/>
+                        <var name="rw_p"/>
+                        <var name="rtheta_p"/>
+                        <var name="rho_p"/>
 			<var name="surface_pressure"/>
 #ifdef DO_PHYSICS
+                        <var name="rainprod"/>
+                        <var name="evapprod"/>
+                        <var name="isltyp"/>
+                        <var name="soilf"/>
+                        <var name="ivgtyp"/>
+                        <var name="landusef"/>
+                        <var name="mminlu"/>
+                        <var name="landmask"/>
+                        <var name="shdmin"/>
+                        <var name="shdmax"/>
+                        <var name="snoalb"/>
+                        <var name="albedo12m"/>
+                        <var name="greenfrac"/>
+                        <var name="lai12m"/>
 			<var name="cldfrac"/>
 			<var name="re_cloud" packages="mp_thompson_in;mp_wsm6_in;mp_tempo_in"/>
 			<var name="re_ice" packages="mp_thompson_in;mp_wsm6_in;mp_tempo_in"/>
 			<var name="re_snow" packages="mp_thompson_in;mp_wsm6_in;mp_tempo_in"/>
+                        <var name="nwfa2d"/>
+                        <var name="nifa2d"/>
+                        <var name="nt_c"/>
+                        <var name="mu_c"/>
+                        <var name="tend_sfc_pressure"/>
+                        <var name="rt_diabatic_tend"/>
+                        <var name="nsteps_accum"/>
+                        <var name="ndays_accum"/>
+                        <var name="tlag"/>
+                        <var name="tday_accum"/>
+                        <var name="tyear_mean"/>
+                        <var name="tyear_accum"/>
+                        <var name="refl10cm"/>
 			<var name="refl10cm_max"/>
 			<var name="rainc"/>
 			<var name="rainnc"/>
+                        <var name="i_rainnc"/>
+                        <var name="rainncv"/>
+                        <var name="snowncv"/>
+                        <var name="snownc"/>
+                        <var name="graupelncv"/>
+                        <var name="graupelnc"/>
+                        <var name="sr"/>
+                        <var name="i_rainc"/>
+                        <var name="cuprec"/>
+                        <var name="raincv"/>
+                        <var name="nca"/>
+                        <var name="cubot"/>
+                        <var name="cutop"/>
+                        <var name="w0avg"/>
+                        <var name="qcg"/>
+                        <var name="sh3d"/>
+                        <var name="sm3d"/>
+                        <var name="qke"/>
+                        <var name="qke_adv"/>
+                        <var name="cov"/>
+                        <var name="qsq"/>
+                        <var name="tsq"/>
+                        <var name="tke_pbl"/>
```

Comment:
I think all of the variables between tke_pbl and det_qv are just diagnostics that shouldn't need to be cycled (for the MYNN). This would remove a lot of 3D variables form the list.

Reply:
Thanks! I will wait for @joeolson42's reply before doing regression tests.

Reply:
Sorry Guoqing, I forgot to share my doc with you. 
(https://docs.google.com/document/d/17fKSS9aZ-_ufOZJfGoGuiPbR5qOPaRXZR3lxEZ_9LMc/edit?tab=t.0, now shared). It has all the variables I think should be removed. I shared this document with other people, but I don't think they have reviewed it yet.

Reply:
@joeolson42 Thanks a lot for sharing the doc!  I agree with all you suggested!!

@Ruifang.Li will conduct a series of retro experiments to remove variables from the `da_state` stream gradually.
I will keep this PR as-is without any further changes. Once Ruifang gets a final list of variables, a new PR will be created to update the `da_state` stream. Thanks!